src/components/cardbuilder/card.css

```diff
@@ -36,6 +36,11 @@ button {
     margin-right: -0.6em;
 }
 
+.itemsContainer.scrollSlider.focuscontainer-x.animatedScrollX {
```

Comment:
@dkanada Maybe we should not use `focuscontainer-x` (and other focusManager-related classes) for styling, since they are used to customize keyboard navigation?

Reply:
Shouldn't be an issue because that section will be removed anyways in favor of the pull request @thornbill has open.